### PR TITLE
Pipelines to use deployment dir path and not single files.

### PR DESCRIPTION
**Pull request by dbogunowicz**

### CODE REVIEW

File: src/deepsparse/transformers/helpers.py [hunk 1]
```diff
@@ -45,18 +43,25 @@
...
+    :return: path to the deployment directory and path to the ONNX file inside
+        the deployment directory
+    """
     if os.path.isfile(model_path):
-        return model_path
+        # return the parent directory of the ONNX file
+        return os.path.dirname(model_path), model_path
 
     if os.path.isdir(model_path):
+        #
```
Comment (dbogunowicz) [inline src/deepsparse/transformers/helpers.py:64]:
```suggestion
```

File: tests/test_data/pipeline_bench_config.json [hunk 2]
```diff
@@ -2,7 +2,6 @@
     "data_type": "dummy",
     "gen_sequence_length": 100,
     "input_image_shape": [500,500,3],
-    "data_folder": "/home/sadkins/imagenette2-320/",
```
Comment (dbogunowicz) [inline tests/test_data/pipeline_bench_config.json:5]: Just doing cleanups (with Sara's blessing), not part of the PR really :)

**Review by bfineran — DISMISSED** [review]
LGTM pending tests and PR description with examples of stubs

**Review by mgoin — APPROVED** [review]
